rerun-failed-run/README.md

```diff
@@ -0,0 +1,42 @@
+### Retrigger Failed Run Action
+
+This composite Github Action can be used to retrigger runs which have failed.
+
+### Usage
+
```

Comment:
Should it also be mentioned that the GitHub Retry App needs to be installed in the target repository?

Reply:
At the moment the app is installed in all repositories in camunda org https://github.com/organizations/camunda/settings/apps/retrigger-gha-workflow/installations

Reply:
ah cool, didn't know - then all good.